Technical-Steering-Committee/issues.md

```diff
@@ -0,0 +1,51 @@
+# Issues
+
+All the squads within the Zowe projects use GitHub to track the issues raised against the projects
+they are responsible for. The issues could be questions, bug reports, request for enhancements and others.
+Every squad responds as outlined below. 
+
+The squads can specify the details of the issue processing for specific project within the project
+repository in the ISSUES.md file in the root of the repository. 
+
+Every squad is expected to triage new issues on the regular squad meeting. The triage process should result 
+into issue categorization: bug, enhancement, question, other and either clarification or issue prioritization: critical, high, medium, low.
+
+The triaging of an issue is done by attaching labels to it (see below for the recommended set of labels):
+
+- new - The issue was not triaged yet 
+
+Types of the issues:
+
+- enhancement - Request to provide new functionality. [The recommended template](https://github.com/zowe/api-layer/blob/v2.x.x/.github/ISSUE_TEMPLATE/feature_request.md) 
+- question - Question around how something works in the specific projects. [The recommended template](https://github.com/zowe/api-layer/blob/v2.x.x/.github/ISSUE_TEMPLATE/question.md)
+- bug - Defect found in one of the existing functionalities within the project. [The recommended template](https://github.com/zowe/api-layer/blob/v2.x.x/.github/ISSUE_TEMPLATE/bug_report.md)
+
+Importance of the issue as perceived by squad organized by importance:
+
+- priority-critical - An issue that needs to be resolved within this PI, ideally within next sprint
```

Comment:
Start tracking PI over PI the information on the bugs and their severity and on enhancements and their Priority. We should add this information to the TSC presentation on PI Planning.

Reply:
Added below.